From cf1d786d5bcdb161db6434cbba285d4c0a3aba24 Mon Sep 17 00:00:00 2001
From: Mirza Krak <mirza.krak@northern.tech>
Date: Fri, 12 Jul 2019 14:38:50 +0000
Subject: [PATCH 1/1] add BOARD argument

This is to be able to override dtb names for other boards then
fsl-imx8mq-evk.dtb

Signed-off-by: Mirza Krak <mirza.krak@northern.tech>

Upstream-Status: Pending

---
 iMX8M/soc.mak | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/iMX8M/soc.mak b/iMX8M/soc.mak
index b24c0db..5efb7c5 100644
--- a/iMX8M/soc.mak
+++ b/iMX8M/soc.mak
@@ -32,6 +32,8 @@ QSPI_HEADER = ../scripts/qspi_header
 QSPI_PACKER = ../scripts/fspi_packer.sh
 endif

+BOARD ?= evk
+
 FW_DIR = imx-boot/imx-boot-tools/$(PLAT)

 $(MKIMG): mkimage_imx8.c
@@ -76,7 +78,7 @@ u-boot-atf-tee.bin: u-boot.bin bl31.bin tee.bin
 clean:
 	@rm -f $(MKIMG) u-boot-atf.bin u-boot-atf-tee.bin u-boot-spl-ddr.bin u-boot.itb u-boot.its u-boot-ddr3l.itb u-boot-ddr3l.its u-boot-spl-ddr3l.bin u-boot-ddr4.itb u-boot-ddr4.its u-boot-spl-ddr4.bin $(OUTIMG)

-dtbs = fsl-$(PLAT)-evk.dtb
+dtbs = fsl-$(PLAT)-$(BOARD).dtb
 u-boot.itb: $(dtbs)
 	TEE_LOAD_ADDR=$(TEE_LOAD_ADDR) ATF_LOAD_ADDR=$(ATF_LOAD_ADDR) ./mkimage_fit_atf.sh $(dtbs) > u-boot.its
 	./mkimage_uboot -E -p 0x3000 -f u-boot.its u-boot.itb
--
2.11.0

